Makefile.due

```diff
@@ -14,7 +14,7 @@ include              $(USART)/Makefile.inc
 
 # settings for Arduino Due projects
 TARGET            ?= arduino_due
-SERIAL_PORT       ?= COM8
+SERIAL_PORT       ?= COM2
```

Comment:
Oh whoops, this should be com port 8 still right?

Reply:
It really does not matter which port it is, in the template, it is port 2.